datadog-opentelemetry/src/lib.rs

```diff
@@ -140,10 +141,27 @@
 //! * [`tracing-opentelemetry`](https://docs.rs/tracing-opentelemetry/0.32.0/tracing_opentelemetry/)
 //!   version: 0.32
 
+// Public re-exports
+pub use core::configuration;
```

Comment:
what if we re-export `configuration::Config` to avoid breaking the STs?

```suggestion
pub use configuration::Config;
pub use core::configuration;
```